spec/unit/puppet_x/chocolatey/chocolatey_common_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+require 'puppet_x/chocolatey/chocolatey_install'
+
+describe 'Chocolatey Common' do
+
+  context ".choco_version" do
+    it "should return PuppetX::Chocolatey::ChocolateyVersion.version" do
+      expected = '0.9.9.0.1'
+      PuppetX::Chocolatey::ChocolateyVersion.expects(:version).returns(expected)
+      PuppetX::Chocolatey::ChocolateyCommon.clear_cached_values
+      PuppetX::Chocolatey::ChocolateyCommon.choco_version.must eq expected
+    end
+  end
+
+  context ".choco_exe?" do
+    it "should be true when version is 0.9.9.0" do
+      PuppetX::Chocolatey::ChocolateyCommon.expects(:choco_version).returns('0.9.9.0')
+      PuppetX::Chocolatey::ChocolateyCommon.clear_cached_values
+      PuppetX::Chocolatey::ChocolateyCommon.choco_exe?.must be_truthy
+    end
+
+    it "should be true when version is greater than 0.9.9.0" do
+      PuppetX::Chocolatey::ChocolateyCommon.expects(:choco_version).returns('0.9.9.0')
```

Comment:
This appears to be a duplicate of the test above.  Is `.returns('0.9.9.0')` correct here?